libr/core/cio.c

```diff
@@ -487,7 +487,7 @@ R_API int r_core_shift_block(RCore *core, ut64 addr, ut64 b_size, st64 dist) {
 	}
 	shift_buf = calloc (b_size, 1);
 	if (!shift_buf) {
-		eprintf ("Cannot allocated %d byte(s)\n", (int)b_size);
+		R_LOG_ERROR ("Cannot allocate %d byte(s)", (int)b_size);
 		return false;
 	}
```

Comment:
`r_return_val_if_fail (shift_buf, false);`

Reply:
i was thinking in adding new r_malloc_fail macros, in 5.8 to let the developers use 3rd party alloc implementations and also unify all the malloc-checks in an unified way. but i dont want those checks to go away in release builds or have invalid asserts at runtime, because those are actual errors that cannot be controlled by the programmer